src/modules/directory.rs

```diff
@@ -74,7 +75,13 @@ pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
     let dir_string = remove_extended_path_prefix(dir_string);
 
     // Apply path substitutions
-    let dir_string = substitute_path(dir_string, &config.substitutions);
+    let dir_string = match substitute_path(dir_string.clone(), &config.substitutions) {
```

Comment:
Is it ok to clone here? Otherwise, the value is moved and I can't move it later (if I understood correctly).